kamailio/default.cfg

```diff
@@ -198,6 +199,12 @@ include_file "antiflood-role.cfg"
 #!ifdef TRAFFIC-FILTER-ROLE
```

Comment:
there is an issue with kamailio defs and the - separator. please use TRAFFIC_FILTER-ROLE
if we later define a TRAFFIC-WHATEVER-ROLE it gets messy